includes/compat/mailchimp-for-woocommerce.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * Newspack setup
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+/**
+ * Disable MC4WC's cookie on Newspack sites to avoid perf issues.
```

Comment:
Perhaps to avoid confusion with the MC4WP plugin, the official name of the Mailchimp for WooCommerce plugin should be used? Here an in function name ("newspack_**mc4c**_remove_cookie")

Reply:
Yeah, good call. I'll update the comments and fix that naming mistake too to make it clearer.